packages/eas-cli/src/commands/branch/list.ts

```diff
@@ -22,7 +22,11 @@ export default class BranchList extends EasCommand {
 
     const projectDir = await findProjectRootAsync();
     const exp = getExpoConfig(projectDir);
-    const projectId = await getProjectIdAsync(exp);
+
+    // TODO(wschurman): this non-interactive flag should be standardized
+    const projectId = await getProjectIdAsync(exp, {
+      nonInteractive: flags['non-interactive'] || flags.json,
```

Comment:
```suggestion
      nonInteractive: flags['non-interactive'],
```

afaik, json cannot be true if non-interactive is false.